### ISSUE TITLE
other pydot forks

### ISSUE THREAD

**Issue by johnyf** [carlos-jenkins/pydotplus#1]
There is a multitude of `pydot` forks. In particular [`nlhelper/pydot`](https://github.com/nlhepler/pydot) appears to be a predecessor of this one. If so, then is there any coordination between this fork and that one? For example, keeping this one and removing the other one.

Another issue concerns the package name. Is there any correspondence with the original author of `pydot`? Since there are developers interested in maintaining `pydot`, it would be good if there was a single fork for the community to contribute to. Even better, if that fork was called `pydot` and was available as such on PyPI.

**Comment by johnyf** [comment]
To clarify, the reason for asking about the original authors is to coordinate. The PyPI package names can be transferred, i.e., the owner can give credentials to other accounts. Another possibility is for the original author to delete the package and @carlos-jenkins or whoever else assumes responsibility of maintenance recreate the package entry.

I would be happy to contact @erocarrera via email and ask about this (or he may respond if a notification reaches him from this post). Over the past year this has been a returning idea to me, however I do not have time to maintain `pydot` myself, so I could not commit to it. From [`twitter/@erocarrera`](https://twitter.com/erocarrera) he appears to be active.

@nlhepler may also have comments on declaring a "main" repository on `github` for `pydot`.

Please let me support why a name change would better be avoided:
1. As you mention, this is a _continuation_ of `pydot`, not a rewrite or a wrapper of some kind. So it should maintain the same name in order to signify this. When I first looked here, I was expecting some rewrite or incompatible refactoring that breaks everything. However, this is not the case. So it will be confusing for people. An example where "plus" denotes something different is [`plyplus`](https://github.com/erezsh/plyplus). In particular, "plus" there means a somewhat higher-level interface.
2. Namespace pollution. By No.1, the motivation for appending a "plus" is syntactic and not semantic. It is not an essential distinction, and this need be reflected on PyPI.
3. Backward compatibility: there is significant amount of code out there that `import pydot` and not `pydotplus`. No reason to change that.

In summary, I hope that logistics won't prevent this to become the maintained version of `pydot`, effectively also transferring the homepage from `googlecode` to `github` (which all major other `python` projects have done long ago).

Thanks for noting it is on PyPI -- I had installed it from there with `pip`.

**Comment by johnyf** [comment]
`pydot` is again under maintenance, at https://github.com/erocarrera/pydot, as noted https://github.com/erocarrera/pydot/issues/119. I will proceed by merging/editing changes from forks, including this one, and make a release.

Version identifiers will distinguish newer from older versions, and avoid confusion. This answers the OP.
